packages/background/src/utils/types/hardware.ts

```diff
@@ -26,4 +27,10 @@ export const HDPaths: DevicesHDPath = {
         { name: 'BIP44 Standard', path: BIP44_PATH, default: true },
         /*{ name: 'Trezor Testnets', path: `m/44'/1'/0'/0` },*/
     ],
+    KEYSTONE: [
```

Comment:
LedgerLive path for keystone. Is this correct? Also, I've seen that you're not injecting hd path in the Keyring controller for keystone devices.

Reply:
But why we don't inject it when adding a new keyring?

https://github.com/block-wallet/extension/blob/756b39ab6c00e7a6dfcfeaa55bcd9b7793323c62/packages/background/src/controllers/KeyringControllerDerivated.ts#L387